department-of-veterans-affairs/next-build
# Add APP_ENV flag for envs

## Description
Relates to #[14977](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14977). (or closes?)


## Testing done
Local testing with mock env files

## Screenshots
<img width="1212" alt="Screen Shot 2023-09-22 at 12 13 10 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/634cad1e-28bf-4bd1-b30a-7255b4b31090">
<img width="1212" alt="Screen Shot 2023-09-22 at 12 15 43 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/268299f0-4fb4-4ea3-bd1b-4e69bc3f0055">


## QA steps
- Run `yarn dev`
  - Verify in network tab that static-entry files are coming from the local generated files
- Create a .env.staging file with a value of  NEXT_PUBLIC_ASSETS_URL=https://s3-us-gov-west-1.amazonaws.com/content.staging.va.gov/generated/ 
  - run `APP_ENV=staging yarn dev'
  - verify in network tab that static-entry files are coming from the appropriate staging bucket

Note:
- This adds the capability to call on different env files with the APP_ENV flag, but still will require that those env files are created in their appropriate environments

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
